index.html

```diff
@@ -5,30 +5,62 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./styles.css" />
+    <link rel="stylesheet" href="./nav-style.css" />
     <title>Document</title>
   </head>
   <body>
     <main class="wrapper">
       <header>
-        <nav></nav>
+        <h2 class="awesome-books">Awesome Books</h2>
+        <nav>
+          <ul class="nav-links">
+            <li id ='booklist' class="link">List</li>
+            <li id ='add-book' class="link">Add new</li>
+            <li id="contactlink" class="link">Contact</li>
+          </ul>
+        </nav>
       </header>
       <section class="hero">
```

Comment:
- [ ] [OPTIONAL] Kindly add the current date to match the wireframe specification and it's in the requirement of the project. You can use the luxon library.